10_code/threshold.py

```diff
@@ -143,72 +148,92 @@
 pop_counties_new[["County", "State"]] = pop_counties_new.NAME.str.split(",", n=1, expand=True)
 pop_counties_new["State"] = pop_counties_new.State.str.strip()
 
-pop_new = pop_counties_new[(pop_counties_new["State"].isin(group_tx_names))]
+pop_new = pop_counties_new[(pop_counties_new["State"].isin(texas_names))]
 
 pop_tx_new = pop_new[["State", "County", "year", "population"]]
 
-pop_tx_new["year"].value_counts() #884 counties ✅
+pop_tx_new["year"].value_counts() 
```

Comment:
one trick to assert here is to multiply the length of county uniques by length of year uniques and see if it's equal to the length of county column. _see the threshold calculation for FL_